.github/ISSUE_TEMPLATE/ask_question.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Ask a question
+about: Use this template to ask general questions about our SDK.
+title: ''
+labels: 'question'
+assignees: ''
+
+---
+
+## Description
+Please clearly state your question. Be as specific as possible to help us understand what you need assistance with. Provide some background on the situation or problem you're facing, explain what you are trying to achieve and where you encountered difficulties.
+
+## Code Snippets (if applicable)
+1. Include any relevant code snippets that illustrate your question or the problem you're experiencing.
+
+2. Highlight the specific lines of code you're unsure about or need help with.
+
+## Expected vs. Actual Results
```

Comment:
This section makes more sense as part of bug report maybe? The way I see `ask a question` is to ask for help with integrating something rather than report unexpected behavior. WDYT?

Reply:
I agree, makes sense.